pdc/apps/release/serializers.py

```diff
@@ -205,3 +207,34 @@ class VariantTypeSerializer(StrictSerializerMixin, serializers.ModelSerializer):
     class Meta:
         model = VariantType
         fields = ('name',)
+
+
+class ReleaseGroupSerializer(StrictSerializerMixin, serializers.ModelSerializer):
+    description         = serializers.CharField(required=True)
+    type                = serializers.SlugRelatedField(slug_field='name',
+                                                       queryset=ReleaseGroupType.objects.all())
+    releases            = ChoiceSlugField(slug_field='release_id',
+                                          many=True, queryset=Release.objects.all(),
+                                          allow_null=True, required=False)
+    active              = serializers.BooleanField(default=True)
+
+    class Meta:
+        model = ReleaseGroup
+        fields = ('name', 'description', 'type', 'releases', 'active')
+
+    def to_internal_value(self, data):
+        releases = data.get('releases', None)
+        if releases:
+            for release in releases:
+                try:
+                    Release.objects.get(release_id=release)
+                except Release.DoesNotExist:
+                    raise serializers.ValidationError({'detail': '%s release does not exist' % release})
+
+        type = data.get('type', None)
```

Comment:
`type` is a builtin function. Maybe using a different name would avoid some confusion. For the class attribute (on model or serializer) it's probably fine, as that will almost always be used with an instance.